python/ql/lib/semmle/python/security/dataflow/SqlInjectionCustomizations.qll

```diff
@@ -9,7 +9,7 @@ private import semmle.python.dataflow.new.DataFlow
 private import semmle.python.Concepts
 private import semmle.python.dataflow.new.RemoteFlowSources
 private import semmle.python.dataflow.new.BarrierGuards
-private import semmle.python.frameworks.SqlAlchemy
+private import semmle.python.Frameworks
```

Comment:
The old import was required for https://github.com/github/codeql/commit/a83bb39d0fa5e1cdd129e6d0927272f5f5f9833f but is no longer needed... no need to have the import here at all now

```suggestion
```